velox/functions/prestosql/window/tests/SimpleAggregatesTest.cpp

```diff
@@ -110,6 +112,29 @@ TEST_P(SimpleAggregatesTest, randomInput) {
   testWindowFunction({makeRandomInputVector(25)});
 }
 
+TEST_F(AggregateWindowTestBase, decimalWindowAggregate) {
+  auto size = 30;
+  auto testAggregate = [&](const TypePtr& type) {
```

Comment:
Nit : Rename method to testDecimal instead ?

Reply:
Updated, thanks.